packages/broker/src/plugins/legacyWebsocket/WebsocketServer.ts

```diff
@@ -87,179 +103,146 @@ export class WebsocketServer extends EventEmitter {
             })
 
         const streams = new StreamStateManager()
-        this.requestHandler = new RequestHandler(streamFetcher, publisher, streams, subscriptionManager, this.metrics, storageNodeRegistry, streamrUrl)
-        networkNode.addMessageListener((msg: Protocol.MessageLayer.StreamMessage) => this._broadcastMessage(msg, streams))
-
-        this._pingInterval = setInterval(() => {
-            this._pingConnections()
-        }, this.pingInterval)
+        this.requestHandler = new RequestHandler(
+            streamFetcher,
+            publisher,
+            streams,
+            subscriptionManager,
+            this.metrics,
+            storageNodeRegistry,
+            streamrUrl
+        )
+        networkNode.addMessageListener((msg: Protocol.MessageLayer.StreamMessage) => this.broadcastMessage(msg, streams))
+
+        this.wss = new WebSocket.Server({
+            server: httpServer,
+            maxPayload: 1024 * 1024
+        })
+        this.wss.on('connection', (ws: WebSocket, request: http.IncomingMessage) => {
+            function closeWithError(internalMsg: string, clientMsg: string) {
+                logger.trace(`rejected connection: ${internalMsg}`)
+                ws.close(1000, clientMsg.slice(0, 100)) // clientMsg has a max size in WS (~123 bytes)
```

Comment:
Is it possible to use the exact `123` value here? Maybe introduce a constant for that.

Reply:
See https://github.com/streamr-dev/network-monorepo/pull/76/commits/5a110d367331f94fb8b61ed7ff7d5e918b202d62

Reply:
fixed